### ISSUE TITLE
Multiple audio questions issue

### ISSUE THREAD

**Issue by larrykind** [globaleaks/globaleaks-whistleblowing-software#3999]
### What version of GlobaLeaks are you using?

4.14.8

### What browser(s) are you seeing the problem on?

Chrome, Firefox, Microsoft Edge

### What operating system(s) are you seeing the problem on?

Windows

### Describe the issue

I set up a questionaire with 4 audio questions (max 60 secs records).
Just want to issue some presumable malfunctioning. Just to note that with one only audio message the issue doesn't seem to happen.

1. On chrome edge and firefox browsers, I often encountered a os (windows) crash when all audio are recorded. A pc reboot was needed.
2. On chrome and firefox browsers, Once client deleted its recording, it's not possible to re-record the audio questions
3. (_also for one only audio message_) If the max recording time expires: the audio player disappears, recording trash button is still on, but audio recording doesn't seem to be there. So a delete and re-record is needed.

### Proposed solution

regarding point n.3:  If this could be the desired behaviour: (i.e this could be the implicit prove that whistoleblower is really recording, and not leaving the player running and recording dummy stuff) I suggest to leave the record as active and reproducible.
Thinking to the case whistleblower is recording an illicit conversation in real time, the wistleblower could not monitor the audio expiring.

**Comment by evilaliv3** [comment]
@larrykind : Would you please clarify more what is happening?

Is this happening only on the the stable release or also on the new client loaded on try.globaleaks.org ?

\cc @msmannan00

**Comment by larrykind** [comment]
@evilaliv3, I verified on 4.14.18, not on try.globaleaks.org. Will try to identify a pattern to reproduce the issue and let you know.

**Comment by msmannan00** [comment]
Yes, the bug is present on multiple audio, it. Also happens for my case created multiple group with audio in older client and it just crashed however it doesn't in new client but will verify it again.

**Comment by msmannan00** [comment]
I had recently tested in different environments and browser. at the moment everything seems to be working with multiple audio questionnaire even under different groups but it seems to be working fine,

The only issue  i faced was after time has expired the stop button disappears, so when submitting report audio should be recorded but thats not the case the audio seems empty on submit

**Comment by larrykind** [comment]
Hi everyone, I did some test ( 4.14.8 ) and I thought I was able to replicate the issue.

0. Tried to put 4 vocal questions on the questionnaire.
1. I noted that the more audio you record the more the cpu is consuming.
2. **the cpu conuming doesn't seem to stop after record is stopped.**  
3. when cpu is 100% system obviously crashes

It happens also with new client but the CPU consumption seems little lower.

**Comment by msmannan00** [comment]
Yes this is an issue, what happens here is that after running multiple recording and not pressing stop while keep recording in new tabs the processor shoots up. when ideally after recording is complete it should stop the processing. however deleting the audio the processing goes to normal. still investigating the possible fix

![Screenshot from 2024-02-20 17-25-16](https://github.com/globaleaks/GlobaLeaks/assets/9531531/a9aae5f7-0a3c-497c-bf06-a06e9c6f00cf)

**Comment by evilaliv3** [comment]
Great!

@msmannan00: Would you please make a patch also for the current AngularJS client ?

**Comment by msmannan00** [comment]
sure will,
